roles/core/log_client/tasks/main.yml

```diff
@@ -42,6 +42,17 @@
   tags:
     - template
 
+- name: Add custom rsyslog files
+  template:
+    src: rsyslog.d_custom.conf.j2
+    dest: "/etc/rsyslog.d/{{ item }}.conf"
+    owner: root
+    group: root
+    mode: 0644
+  loop: "{{ log_client_custom_conf }}"
+  tags:
+    - custom_conf
```

Comment:
@oxedions should the tag be 'template' here?

Reply:
Yes please 🙂

Reply:
It will be the same as the previous task, is it not a problem ?

Reply:
Both are templates, so it is ok.